### DISCUSSION TITLE
Support older versions of react

### DISCUSSION THREAD

**Discussion by honzahruby** [Omnistac/zedux#44] [Ideas]
Have you considered using the official shim for useSyncExternalStore (https://www.npmjs.com/package/use-sync-external-store) that would allow you to support react 16 and 17?

**Comment by bowheart** [comment]
Absolutely. Thanks for bringing it up. You are the first, so we hadn't considered it up to this point.

I'll try plugging it in today if I get a minute. If tests pass, I don't see any reason why we can't release a patch for this very soon.

**Reply by honzahruby** [reply]
That would be perfect, there are probably many people who haven’t yet upgraded to the latest version. Thank you!

**Reply by bowheart** [reply]
Update: Plugged it in and all tests pass. PR here: #45

The shim adds over 1kb to the minified build, so I'm gonna leave the PR up for a day or two to see if anyone has thoughts on that.

**Comment by bowheart** [comment]
This has been merged to master in #45 and released in [v1.0.1](https://github.com/Omnistac/zedux/releases/tag/v1.0.1). Closing
